pyhomematic/devicetypes/thermostats.py

```diff
@@ -21,7 +19,14 @@ def __init__(self, device_description, proxy, resolveparamsets=False):
         self.BOOST_MODE = 3
         self.COMFORT_MODE = 4
         self.LOWERING_MODE = 5
-        self.OFF_VALUE = 4.5
+        if "HMIP" in self.TYPE.upper():
+            self.OFF_VALUE = self._proxy.getParamset("%s:1" % self._ADDRESS, "MASTER").get("TEMPERATURE_MINIMUM", 4.5)
+            self.ON_VALUE = self._proxy.getParamset("%s:1" % self._ADDRESS, "MASTER").get("TEMPERATURE_MAXIMUM", 30.5)
+        else:
```

Comment:
Do that IO?

Reply:
Yes. But other devices do that as well: https://github.com/danielperna84/pyhomematic/blob/021eda71c97dcf21de89c3c74e8b1c05479570ef/pyhomematic/devicetypes/actors.py#L251
During startup IO is normal. There's no IO after the init with this solution.

Reply:
Our Issue is that we hit the xmlrpc on startup with a lot of requests. Do we receive all this details and the current state also on initial call? So we can use this data on startup and don't need get all this data again on HA and we don't run into issue of all this error inside logs from startup

Reply:
Lieber auf deutsch:
Ich denke die CCU hält diese paar Requests aus. Ja, sind jetzt schon nicht wenige, aber ich sehe keinen anderen Weg. Das sind Konfigurationsdaten die im MASTER Paramset liegen. Die kriegen wir nicht anders als aktiv danach zu fragen. Das wird auch nicht über Events übermittelt. Wenn man also von HASS aus die tatsächlich konfigurierten Werte nutzen will, dann muss man das meines Erachtens nach so machen.  
Ich denke aber wie gesagt, dass die CCU das aushält. Selbst mit 10 Thermostaten dürfte das nicht mehr als eine Sekunde kosten um die extra Requests zu verarbeiten.